### ISSUE TITLE
Saleor CLI can't be run with `--no-warnings` flag

### ISSUE THREAD

**Issue by prajogoatmaja** [saleor/saleor-cli#279]
# Environment
WSL Ubuntu 20.04.3 LTS 
Saleor Commerce CLI v1.13.1 
nvm v0.39.1
npm v8.18.0
Node.js v16.17.0

# Context
After following the tutorial from [https://learn.saleor.io/cli/getting-started/](https://learn.saleor.io/cli/getting-started/), I failed to run the command  `saleor`, with the response error

> `/usr/bin/env: ‘node --no-warnings’: No such file or directory `

I found no bugs related to symbolic link on my side, so I tried to fix the problem myself by removing the `--no-warnings` flag located in `build/cli.js` shebang inside the `node_modules` folder. It did work, but is this an expected behavior?

# Fix
Removing the  `--no-warnings` flag located in `src/cli.ts` shebang

# Request
If this fix is acceptable, may i create the pull request for this?

**Comment by petar-stefanov** [comment]
Same issue here, after `npm i -g @saleor/cli` and running `saleor` i get an error:
 
> ```
>  /usr/bin/env: ‘node --no-warnings’: No such file or directory
>  /usr/bin/env: use -[v]S to pass options in shebang lines
> ```

Ubuntu 22.04.1 LTS
npm v8.18.0
Node.js v16.17.0

**Comment by kamenchunathan** [comment]
I encountered the same error 

```
 /usr/bin/env: ‘node --no-warnings’: No such file or directory
 /usr/bin/env: use -[v]S to pass options in shebang lines

```
Setup:
20.04.1-Ubuntu  x86_64 GNU/Linux
npm v8.16.0
node vv16.16.0

This is fixed by adding the -S flag to the shebang of cli.js to be able to pass parameters to node
`#!/usr/bin/env -S node --no-warnings`

**Comment by kamenchunathan** [comment]
It looks like they tried to fix this in the commit 2da8b193f6f9eb8aef6c09e536d1f8d50fe5fd95 but running the script with this shebang does not start and apparently runs in an infinite loop
The problem is that only one argument can be passed in a shebang therefore the -S flag is still necessary 
This is explained here [https://bugs.launchpad.net/ubuntu/+source/coreutils/+bug/1421760]( https://bugs.launchpad.net/ubuntu/+source/coreutils/+bug/1421760)

**Comment by zaiste** [comment]
@prajogoatmaja @kamenchunathan @petar-stefanov this is fixed in `1.15`. You can update it with:
```
npm i -g @saleor/cli@latest
```
